test/gst-vaapi/encode/avc.py

```diff
@@ -46,9 +46,11 @@ def test_cqp(case, gop, slices, bframes, qp, quality, profile):
   if mprofile is None:
     slash.skip_test("{} profile is not supported".format(profile))
 
+  hwup_format = mapformat_hwup(params["format"])
   params.update(
     gop = gop, slices = slices, bframes = bframes, qp = qp, quality = quality,
-    profile = mprofile, mformat = mapformat(params["format"]))
+    profile = mprofile, mformat = mapformat(params["format"]),
+    hwup_format = hwup_format)
```

Comment:
Please inline `hwup_format = mapformat_hwup(params["format"])` here.  Same for all other tests.

Reply:
inlined, thanks